File: ContactBook.Web/contactbook.vue/src/models/ContactPhone.ts
export default class phones
{
    id:number;
    contactId:number;
    phoneNumber:string;
    type:number;

    constructor(id:number,contactId:number,phoneNumber:string,type:number)
    {
        this.id = id,
        this.contactId = contactId,
        this.phoneNumber = phoneNumber,
        this.type = type
    }
}
export enum PhoneType{
    Home,
    Work,
    MobilePhone
}

File: ContactBook.core/Models/ContactPhone.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ContactBook.core
{
    
    public class ContactPhone
    {
        public int Id { get; set; }
        public int ContactId { get; set; }
        public string PhoneNumber { get; set; }
        public PhoneType Type { get; set; }
        public virtual Contact contact { get; set; }
    }
    public enum PhoneType
    {
         Home,
         Work,
         MobilePhone
    }
}

